pkg/aggregator/time_sampler_worker.go

```diff
@@ -49,6 +57,11 @@ type dumpTrigger struct {
 	done chan error
 }
 
+type blocklistTrigger struct {
+	trigger
```

Comment:
Since the receiver does not use any of the `trigger` fields, would it make sense to send just `*utilstrings.Blocklist`, instead of creating a new struct?

Reply:
Adressed.